### ISSUE TITLE
CSRF Token Expired 

### ISSUE THREAD

**Issue by rojinpmani** [prashants/webzash#99]
Hi SIr,

I installed script in localhost and when i try to delete ledger, I am getting these errors. Let me know why?. 

Error:
CSRF Token Expired ! You have exceeded the time limit set to complete a request, so for security reasons you need to redo whatever you were trying to do.

Hope to hear from you soon.

**Comment by prashants** [comment]
Did it work after retry ?

The session is getting timed out.

**Comment by rojinpmani** [comment]
Yes, I tried several times after refresh and reload page. Besides, There are a few java-script error finds in console.
